sunxspex/emission.py

```diff
@@ -578,14 +541,14 @@ def split_and_integrate(*, model, photon_energies, maxfcn, rerr, eelow, eebrk, e
     # Part 1, below en_val[0] (usually eelow)
     if model == 'thick-target':
         if P1.size > 0:
-            print('Part1')
+            logging.debug('Part1')
```

Comment:
is there any argument besides legacy for keeping the code in 3 very similar parts like this?

Reply:
No just not in scope of for this PR

Reply:
okay. so is that also why nothing has been done about #58 ?